src/format/opamTypes.mli

```diff
@@ -238,6 +238,7 @@ type 'a concrete_action = [
   | 'a atomic_action
   | `Build of 'a
   | `Fetch of 'a
+  | `Fetches of 'a * 'a list
```

Comment:
```suggestion
  | `Fetch of 'a * 'a list
```
Any particular reason you kept the singular `Fetch` action?

Reply:
Hm yes could be a good idea; depends how inconvenient it is in the cases where you used to match on the `Fetch` action and would have to assert that the list is empty. Are there even any such cases ?